sms_939/models/sms_sponsorship.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    @author: Théo Nikles <theo.nikles@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class SponsorshipSMS(models.Model):
+    _name = 'sms_sponsorship'
```

Comment:
sms sponsorship is not a good name, because it has nothing to do with sponsorships. It's only sms logs so I would propose this name instead: `sms.log`. Please rename all needed objects.

Reply:
Done.